scripts/setup-tf-operator.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to build a cluster and create a namespace from our
+# argo workflow
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME="${CLUSTER_NAME}"
+REGION="${AWS_REGION:-us-west-2}"
+REGISTRY="${ECR_REGISTRY:-527798164940.dkr.ecr.us-west-2.amazonaws.com}"
+VERSION="${PULL_BASE_SHA}"
+GO_DIR=${GOPATH}/src/github.com/${REPO_OWNER}/${REPO_NAME}
+
+echo "Configuring kubeconfig.."
+aws eks update-kubeconfig --region=${REGION} --name=${CLUSTER_NAME}
+
+echo "Update tf operator manifest with new name and tag"
+cd manifests/
+kustomize edit set image gcr.io/kubeflow-images-public/tf-operator=${REGISTRY}/${REPO_NAME}:${VERSION}
```

Comment:
@ChanYiLin This means we need you to provide a list of ECR registry you want to use, otherwise you won't be able to create ECR registries during testing.

E.g, 

```
527798164940.dkr.ecr.us-west-2.amazonaws.com/katib/v1beta1/earlystopping-medianstop
527798164940.dkr.ecr.us-west-2.amazonaws.com/katib/v1beta1/file-metrics-collector
.....
```